<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ALBA_Actuator</class>
 <widget class="QWidget" name="ALBA_Actuator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>117</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InOut</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="actuatorBox">
     <property name="title">
      <string>InGroup</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="stateLabel">
          <property name="text">
           <string>state</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="cmdInButton">
            <property name="text">
             <string>In</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cmdOutButton">
            <property name="text">
             <string>Out</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
